features/mbedtls/targets/TARGET_CRYPTOCELL310/ecdsa_alt.c

```diff
@@ -0,0 +1,325 @@
+/*
+ *  ecdsa_alt.c
+ *
+ *  Copyright (C) 2018, ARM Limited, All Rights Reserved
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "mbedtls/ecdsa.h"
+#include <string.h>
+#include "crys_ecpki_ecdsa.h"
+#include "crys_ecpki_build.h"
+#include "crys_common.h"
+#include "crys_ecpki_kg.h"
+#include "crys_ecpki_domain.h"
+#include "crys_ec_edw_api.h"
+#include "mbedtls/platform.h"
+#include "cc_internal.h"
+
+/* Implementation that should never be optimized out by the compiler */
+static void mbedtls_zeroize( void *v, size_t n ) {
+    volatile unsigned char *p = (unsigned char*)v;
+    while( n-- ) *p++ = 0;
+}
```

Comment:
This function is repeated across several files. I do not think there will be problems when the new version of Mbed TLS that include mbedtls_platform_zeroize() is used, but mbedtls_zeroize should still be removed.

Reply:
I know. It will be removed when `mbedtls_platform_zeroize()` is introduced.
See https://github.com/ARMmbed/mbed-os-cc-porting/pull/7#discussion_r183620110